client/components/report-data-labels/labels.directive.js

```diff
@@ -0,0 +1,36 @@
+angular.module('probedock.reportDataLabels').directive('reportDataLabels', function() {
+  return {
+    restrict: 'E',
+    controller: 'ReportDataLabelsCtrl',
+    templateUrl: '/templates/components/report-data-labels/report-data-labels.template.html',
+    replace: true,
+    scope: {
+      report: '=',
+      organization: '=',
+      nbProjectsToShow: '=?',
+      projectFilter: '=?',
+      showVersionsOnly: '=?'
```

Comment:
Naming not consistent with `versionOnly` of the `projectVersionLabel` directive.